src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,157 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final Object[] DEFAULTCAPACITY_EMPTY_ELEMENTDATA = {};
+    private static final Object[] EMPTY_ELEMENTDATA = {};
+    private static final int NEGATIVE_INDEX_MARKER = -1;
+    private static final double CAPACITY_GROWTH_RATIO = 1.5;
+    private Object[] elementData;
+    private int size;
+
+    public ArrayList(int initialCapacity) {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the constructor declaration.